index.html

```diff
@@ -0,0 +1,285 @@
+            </div>
+
+            <!-- My recent work -->
+            <div class="My-Recent-Work">
+              <h3>MY RECENT WORK</h3>
+            </div>
+          
+            <!-- line3 -->
+            <div class="hr" style="padding:20px; margin: 20px; color: dark">
+              <hr>
+            </div>
+
+            <!--PLACE HOLDER IMAGE-->
+            <div class="place-holder-img">
+              <img src="./img/ImgPlaceholder.png" alt="place holder">
+            </div>
+
+            <!-- MUTLI POST STORIES -->
+            <div class="title-post">
+              <h2>Multi-Post Stories</h2>
+            </div>
+            <div class="paragraph2">
+              <p>A daily selection of privately <br>
+                 personalized reads; no accounts <br>
+                 or sign-ups required. has been the <br>
+                 industry's standard dummy text <br>
+                 ever since the 1500s, when an <br>
+                 unknown printer took a standard <br>
+                 dummy text.</p>
+            </div>
+
+            <!-- languajes de programacion -->
+            <ul class="programming-languages">
+              <li><a href="">CSS</a></li>
+              <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+              <li><a href="">html</a></li>
+              <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+              <li><a href="">Ruby</a></li> 
+            </ul>
+
+            <!-- SEE PROJECT BUTTON -->
+            <div class="see-project">
+              <img src="./img/Enable.png"alt="see project"></div>
+     </section>
+
+
+          <!--PLACE HOLDER IMAGE2-->
+          <section>
+        <div class="place-holder-img">
+       <img src="./img/ImgPlaceholder.png" alt="place holder">
+      </div>
+      <!-- MUTLI POST STORIES -->
+      <div class="title-post">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="paragraph2">
+        <p>A daily selection of privately <br>
+           personalized reads; no accounts <br>
+           or sign-ups required. has been the <br>
+           industry's standard dummy text <br>
+           ever since the 1500s, when an <br>
+           unknown printer took a standard <br>
+           dummy text.</p>
+      </div>
+
+        <!-- languajes de programacion -->
+        <ul class="programming-languages">
+        <li><a href="">CSS</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+        <li><a href="">html</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+        <li><a href="">bootstrap</a></li>
+        <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
+        <li><a href="">Ruby</a></li> 
+      </ul>
```

Comment:
- [ ] [OPTIONAL] The `<a>` tag is not necessary for the languages list since these are not links